MyPerfectOnboardingApplication/Sources/MyPerfectOnboarding.Contracts/Services/ListItem/IAdditionService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using MyPerfectOnboarding.Contracts.Models;
+
+namespace MyPerfectOnboarding.Contracts.Services.ListItem
```

Comment:
Consider plural here, for one to avoid class name collision and also to be on par with `Generators` plural:
```suggestion
namespace MyPerfectOnboarding.Contracts.Services.ListItems
```